src/components/collapsibleCard/view/collapsibleCardView.js

```diff
@@ -50,6 +50,7 @@ class CollapsibleCardView extends PureComponent {
     Animated.timing(this.anime.height, {
       toValue: this.anime.expanded ? this._getMinValue() : this._getMaxValue() + (moreHeight || 0),
       duration: 200,
+      useNativeDriver: false,
```

Comment:
using native driver as false means js thread will be handling the animation. can you please try replacing Animated package with `react-native-reanimated` as we discussed. Start with this one and better replace the package in all places.

Reply:
@noumantahir height property is not supported animation on native thread. Tried to find a solution, one solution which I found is to replace height with scaling because scaling supports native animation. I can give that a try again

Reply:
replaced with react-native-reanimated